wren-ai-service/src/providers/document_store/qdrant.py

```diff
@@ -17,9 +17,13 @@
 from qdrant_client.http import models as rest
 
 from src.core.provider import DocumentStoreProvider
-from src.providers.llm.openai import EMBEDDING_MODEL_DIMENSION
 from src.providers.loader import provider
 
+if os.getenv("LLM_PROVIDER") == "ollama":
+    from src.providers.llm.ollama import EMBEDDING_MODEL_DIMENSION
+else:
+    from src.providers.llm.openai import EMBEDDING_MODEL_DIMENSION
+
```

Comment:
I'm curious about the azure openai having a different default dimension, should we also add it here?